src/transactions/OperationFrame.h

```diff
@@ -62,6 +76,11 @@ class OperationFrame
     LedgerTxnEntry loadSourceAccount(AbstractLedgerTxn& ltx,
                                      LedgerTxnHeader const& header) const;
 
+    bool preloadEntryHelper(
```

Comment:
Do we actually use this anywhere?

Reply:
No this was from an earlier iteration where I was trying to share some code, but it was getting messy.